Userland/Libraries/LibWeb/CSS/Default.css

```diff
@@ -31,6 +31,8 @@ input:not([type=submit], input[type=button], input[type=image], input[type=reset
     min-height: 16px;
     cursor: text;
     overflow: hidden;
+
+    background-color: Field;
```

Comment:
Does `color: FieldText` work OK here too? Those are supposed to be used in pairs together.

Reply:
Seems fine to me, added